src/hooks/useENSRegistration.ts

```diff
@@ -118,9 +118,15 @@ export default function useENSRegistration({
       {}
     );
 
+    const recordKeysWithValue = (Object.keys(
+      records
+    ) as (keyof Records)[]).filter((key: keyof Records) => {
+      return Boolean(records[key]);
+    });
+
     const keysToRemove = differenceWith(
       Object.keys(initialRecords),
-      Object.keys(records),
+      recordKeysWithValue,
```

Comment:
added this to detect a record key that's being deleted with an empty value, before we weren't taking in account record values that are being manually deleted character by character in keyboard instead of pressing the record button https://recordit.co/kCHyl1MpSp